### PR TITLE
[CDAP-7463] Changes footer for CDAP, Tracker, and Hydrator

### PR DESCRIPTION
**Pull request by PGuimarais**
tracker

hydrator

cdap footer

JIRA:
https://issues.cask.co/browse/CDAP-7463

Bamboo Build:
http://builds.cask.co/browse/CDAP-DRC4723

### CODE REVIEW

File: cdap-ui/app/cdap/components/Footer/index.js [hunk 1]
```diff
@@ -31,28 +31,11 @@ export default class Footer extends Component {
       <footer>
         <div className="container">
           <div className="row text-muted">
-            <div className="col-sm-6 text-uppercase">
-              <p>
-                <span className="cask-logo"></span>
-                <span>Copyright &copy; {this.copyrightYear} Cask Data, Inc.</span>
+            <div className="text-uppercase">
+              <p className="text-center">
+                <span>Copyright &copy; 2011-{this.copyrightYear} Cask Data, Inc.</span>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Footer/index.js]: @PGuimarais did you confirm we need the year 2011? I think all we need is the current year

Comment (PGuimarais) [reply cdap-ui/app/cdap/components/Footer/index.js]: This was from the design. Can discuss.

**Comment by ajainarayanan** [comment]
On smaller screen size in hydrator studio alone this happens. I think we just need to remove the rule as the previous footer's size increased on a smaller size I believe.

screen shot 2016-10-27 at 1 38 55 pm

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**